pmm/users.tf

```diff
@@ -0,0 +1,23 @@
+# TODO we need to use roles instead users
+resource "aws_iam_user" "nikita" {
+    name = "nikita.beletskii-cli"
+}
+
+resource "aws_iam_access_key" "nikita" {
+  user = aws_iam_user.nikita.name
+}
+
+data "aws_iam_user" "mykola" {
+  user_name = "mykola-cli"
+}
```

Comment:
This must be an example, right? If so, let's maybe comment it out?

Reply:
We already have this user :)